examples/mysql2.rb

```diff
@@ -0,0 +1,53 @@
+# example of
+# 1. Connecting to a mysql database server
+# 2. Creating five databases on that server
+# 3. Creating five tables on each of those databases
+# 4. Populating each of those five tables with some dummy data
+# 5. Printing that data out
+# 6. Dropping the datbases
+require 'sequel'
+Sequel.extension :migration, :core_extensions
+
+DB = []
+
+# Just connect to the database server, not a particular DB on the server
+DB[0] = Sequel.connect('mysql://root:toor@testing-mysql')
+
+[1,2,3,4,5].each do |n|
+  puts "Dropping database #{n} on testing-mysql"
+  DB[0].run "DROP DATABASE IF EXISTS db#{n}"
+
+  puts "Creating database #{n} on testing-mysql server"
+  DB[0].run "CREATE DATABASE IF NOT EXISTS db#{n};"
+
+  puts "Using database #{n} on testing-mysql"
+  DB[n] = Sequel.mysql("db#{n}",
+                       :user => 'root',
+                       :password => 'toor',
+                       :host => 'testing-mysql')
+
+  [11,12,13,14,15].each do |m|
```

Comment:
They are used to create the tables `table_m` on each database.
I'll add a comment about this.